src/components/ReportIssuePopover.vue

```diff
@@ -1,8 +1,8 @@
 <template>
   <ion-content>
     <ion-list>
-      <ion-item v-for="reason in rejectReasons" :key="reason.enumId" @click="updateIssue(reason.enumId)" button>
-        {{ reason.description ? translate(reason.description) : reason.enumId  }}
+      <ion-item v-for="reason in rejectReasonOptions" :key="reason.enumId" @click="updateIssue(reason.enumId)" button>
+        {{ reason.description ? translate(reason.description) : reason.enumDescription ? translate(reason.enumDescription) : reason.enumId  }}
```

Comment:
Same as the comment in BOPIS app regarding description.

Reply:
This will not give issue, since we are not fetching description field in case of EnumerationGroupAndMember, so we won't have description and EnumDescription at the same time.